File: Back/BackEnd/BackEnd/DTO/UsuarioAmigosDTO.cs
﻿using BackEnd.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.DTO
{
    public class UsuarioAmigosDTO : IUsuarioAmigosDTO
    {
        public Int64 IDAmigo { get; set; }
        public string Usuario_Nome { get; set; }
        public decimal Posicao_Latitude { get; set; }
        public decimal Posicao_Longitude { get; set; }
    }
}

File: Front/localizaFront/src/app/DTO/usuario-amigos-dto.ts
import { IUsuarioAmigosDTO } from '../Interface/iusuario-amigos-dto';

export class UsuarioAmigosDTO implements IUsuarioAmigosDTO {
    iDAmigo: number;
    usuario_Nome: string;
    posicao_Latitude: number;
    posicao_Longitude: number;
}
